main/src/ca/uwaterloo/flix/language/ast/shared/SymUse.scala

```diff
@@ -37,7 +38,7 @@ object SymUse {
   /**
     * Represents a use of an effect sym.
     */
-  case class EffectSymUse(sym: Symbol.EffectSym, loc: SourceLocation)
+  case class EffectSymUse(sym: Symbol.EffectSym, qname: QName, loc: SourceLocation)
```

Comment:
I think - given the earlier confusion - each parameter should be documented.

Reply:
Also consider whether loc is redundant

Reply:
Both are good points. For the second point, if we do remove `loc` maybe it would be useful to add a `loc` function that just accesses `qname.loc`, just to keep the pattern of AST nodes having their SourceLocation accessible through `.loc`?